apps/backend/src/lambdas/helpers.ts

```diff
@@ -0,0 +1,27 @@
+import { parse } from 'date-fns';
+/*eslint-disable */
+export const parseSNSMessage = (parsedMessage: any) => {
+  const from_date = parse(
+    parsedMessage.Records[0].Sns.Message.period.from,
+    'yyyy-MM-dd',
+    new Date()
+  );
+  console.log(from_date);
+  const to_date = parse(
+    parsedMessage.Records[0].Sns.Message.period.to,
```

Comment:
Do we need to do some type checking around this function? e.g. at the start, check that parsedMessage.records?.length > 0, and that the necessary fields exist `Sns.Message.period.to`